contracts/utils/Operators.sol

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.9;
+
+import { IOperators } from '../interfaces/IOperators.sol';
+import { Ownable } from './Ownable.sol';
+
+/**
+ * @title Operators
+ * @dev Contract module which provides an access control mechanism, where
+ * there is an account (an owner) that can be granted exclusive access to
+ * specific functions, and an operator that can execute arbitrary functions
+ * on any smart contract.
+ *
+ * The owner account is initially set as the deployer address.
+ */
+contract Operators is Ownable, IOperators {
+    mapping(address => bool) public operators;
+
+    /**
+     * @dev Sets the deployer as the initial owner.
+     */
+    constructor() {
+        address initialOwner = msg.sender;
+        // solhint-disable-next-line no-inline-assembly
+        assembly {
+            sstore(_OWNER_SLOT, initialOwner)
+        }
+    }
+
+    /**
+     * @dev Modifier that requires the "msg.sender" to be an operator.
+     * Emits a NotOperator error if the condition is not met.
```

Comment:
```suggestion
     * Reverts with a NotOperator error if the condition is not met.
```
perhaps, unless emit is also used in this context

Reply:
reverts with sounds better